src/Operation/Watch.php

```diff
@@ -353,10 +351,8 @@ private function createChangeStreamIterator(Server $server): ChangeStreamIterato
      *
      * The command will be executed using APM so that we can capture data from
      * its response (e.g. firstBatch size, postBatchResumeToken).
-     *
-     * @return CursorInterface&Iterator
      */
-    private function executeAggregate(Server $server)
+    private function executeAggregate(Server $server): CursorInterface
     {
         addSubscriber($this);
```

Comment:
Nothing relevant to this PR, but I want to point out that it's a bit awkward that we can assign subscribers to Managers but not individual Servers. That forces us to use a global subscriber here.

I suppose we could add `Server::getManager()` to work around that :man_shrugging: